swagger.yaml

```diff
@@ -3564,6 +3564,59 @@ paths:
           description: Error.
           schema:
             $ref: '#/definitions/error'
+    post:
+      tags:
+        - events
+      security:
+        - agentAuth: []
+      description: Add new assisted installer event.
+      operationId: v2AddEvent
```

Comment:
Maybe EmitEvent is more descriptive than Add?  (Add to what?  At first I thought it was defining a new type.)